internal/driver/deviceclient.go

```diff
@@ -72,7 +72,10 @@ func createCommandInfo(req *models.CommandRequest) (*CommandInfo, error) {
 			return nil, errors.NewCommonEdgeX(errors.KindLimitExceeded, fmt.Sprintf("register size should be within the range of 1~123, get %v.", length), nil)
 		}
 	} else {
-		length = calculateAddressLength(primaryTable, rawType)
+		length, err = calculateAddressLength(primaryTable, rawType)
+		if err != nil {
+			return nil, err
```

Comment:
```suggestion
			return nil, errors.NewCommonEdgeXWrapper(err)
```

this is the way to record the error calling line for the future debug